package.json

```diff
@@ -9,27 +9,34 @@
     "src/assets/scss/search.scss"
   ],
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "test:unit": "vue-cli-service test:unit",
-    "lint": "vue-cli-service lint",
-    "lint:fix": "vue-cli-service lint --fix",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "build:lib": "vue-cli-service build --name lib --target lib --dest lib src/lib-setup.js"
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
```

Comment:
change this to `preview`,  otherwise switching from webpack -> vite, people will run npm run serve and wonder why their page wont update